src/Home/Home.spec.tsx

```diff
@@ -37,6 +39,32 @@ jest.mock("../More/useApplicationInfo", () => {
 jest.mock("./useBluetoothStatus.ts")
 
 describe("Home", () => {
+  it("allows users to share the application", () => {
```

Comment:
This is an example on how we can improve testing logic dependant on the values from the environment